examples-gallery/beginner/plot_wheel_on_bumpy_road.py

```diff
@@ -0,0 +1,425 @@
+r"""
+Quarter Car Model on a Bumpy Road
+=================================
+
+Objective
+---------
+
+- Show on a simple example how to simultaneously optimize free
+  parameters and unknown trajectories of a system.
+- Show how one can iterate from a simpler problem (road less bumpy) to a
+  harder problem (road more bumpy).
+- Show how to use an additional state variable to get the acceleration of a
+  body into the objective function.
+- Show how to avoid possible unwanted unknown trajectories ( here
+  :math:`\dfrac{d^2}{dt^2}x_{car}`).
+
+
+Introduction
+------------
+
+A quarter car model must move from a starting point to a final point on a bumpy
+road. The body is connected to the wheel on the road by a linear spring and a
+damper.
+Body and wheel are modeled by particles. Movement is in the X/Z plane. Gravity
+points in the negative Z direction.
+
+I took the tire stiffness from here:
+
+https://kktse.github.io/jekyll/update/2021/07/18/re71r-255-40-r17-tire-vertical-stiffness.html
+
+
+**States**
+
+- :math:`x_{car}` : x position of the car [m]
+- :math:`z_{car}` : z position of the car [m]
+- :math:`z_{wheel}` : z position of the wheel [m]
```

Comment:
```suggestion
- :math:`z_\textrm{wheel}` : z position of the wheel [m]
```

Words in variable names should not be italicized text.